source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,145 @@
+.. _csharp-aggregation:
+
+===========
+Aggregation
+===========
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code example, transform, pipeline
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to perform
+**aggregation operations**.
```

Comment:
S: redo this section to align more with the page structure on the [Rust aggregation page](https://github.com/mongodb/docs-rust/blob/master/source/fundamentals/aggregation.txt)

That page has a better organization of information and the copy has been updated to align more with the style guide